src/lib/RegionList.svelte

```diff
@@ -10,10 +10,31 @@
 </script>
 
 <!-- TODO add filter by name and filter by non-country (aka ISO2===null) -->
+<h1 class="text-lg">Regions</h1>
```

Comment:
Is `Regions` the best name to use for 'Non-country regions'? Also for file naming, API, et cetera might be confusing; when does 'region' refer to any region (incl countries) and when does it refer to a group of countries?

Perhaps 'Collectives' or "International Groups" or "Global Alliances"?

Reply:
Currently non-country regions are called regions.
Peter has some alternate naming suggestion: 'Collectives' or "International Groups" or "Global Alliances"
Some more suggestions 'Geopolitical entities', 'Regional blocs', 'Supranational ...', ' Intergovernmental  ...'

@MarkMDekker what is your preference?

Reply:
Let's keep this as is for now and possibly improve later on. Don't want to hold up the merge.